libs/ui-lib/lib/ocm/components/featureSupportLevels/featureStateUtils.ts

```diff
@@ -113,13 +118,18 @@ const getCnvDisabledReason = (
 const getLvmDisabledReason = (
   activeFeatureConfiguration: ActiveFeatureConfiguration,
   isSupported: boolean,
+  platformType?: PlatformType,
 ) => {
   if (!activeFeatureConfiguration) {
     return undefined;
   }
   const operatorLabel = isSupported ? LVMS_OPERATOR_LABEL : LVM_OPERATOR_LABEL;
   if (!isSupported) {
-    return `${operatorLabel} is not supported in this OpenShift version.`;
+    if (platformType === 'nutanix') {
```

Comment:
Is the fact that Nutanix is selected what makes the feature unsupported?
If that's the case, then the condition should be outside the check for `isSupported`

(Since IIUC, `isSupported` comes from checking the feature support levels which don't take into consideration the selected operators or selected platform)

Reply:
I put the condition outside isSupported check.